sync3/handler/connstate.go

```diff
@@ -473,6 +473,8 @@ func (s *ConnState) getInitialRoomData(ctx context.Context, roomSub sync3.RoomSu
 		var inviteState []json.RawMessage
 		// handle invites specially as we do not want to leak additional data beyond the invite_state and if
 		// we happen to have this room in the global cache we will do.
+		// Furthermore, rooms we have been invited to for the first time ever will not be in the global cache yet,
```

Comment:
who is "we"---the proxy as a whole, or the user in question? (I think the former?)

Reply:
The former, clarified.